src/store/imageBuilderApi.ts

```diff
@@ -721,6 +721,7 @@ export type PackagesResponse = {
   data: Package[];
 };
 export type DistributionProfileItem =
+  | "None"
```

Comment:
Hey, just a quick comment on this one:

One thing to remember is that the api files shouldn't get updated manually. These files are generated by the `npm run api` command. The command is loading the openapi yaml file for the image builder api and generating the types for us. If you change manually the file, the next time someone will update the yaml file, your changes will be erased and lost.

And the other thing to remember is that the openapi yaml file in the frontend *has to* be in sync with image builder's file that lives in the image builder repository too.

If you feel like a None option is necessary, the workflow is the following:
* You need to first issue a PR on image builder to have its API changed.
* Once it's merged, import the new yaml file into the frontend repository.
* Run the `npm run api` command
* And issue a PR on the frontend :slightly_smiling_face:

Reply:
BTW, I don't think adding a `None` option to the image builder api is a good idea. In the API perspective, not sending a profile means `None` already.

Reply:
I remove it from the imageBuilderApi. thank you